test/metadata/testMetaDataAPI.js

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright 2020 The caver-js-ext-kas MetaData
+ *
+ * Licensed under the Apache License, Version 2.0 (the “License”);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an “AS IS” BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const fs = require('fs')
+const fpath = require('path')
+const chai = require('chai')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+const chaiAsPromised = require('chai-as-promised')
+
+chai.use(chaiAsPromised)
+chai.use(sinonChai)
+
+const expect = chai.expect
+
+const CaverExtKAS = require('../../index.js')
+
+let caver
+const { url, chainId, accessKeyId, secretAccessKey, operator } = require('../testEnv').auths.metaDataAPI
+
+const sandbox = sinon.createSandbox()
+
+describe('MetaData API service enabling', () => {
+    beforeEach(() => {
+        caver = new CaverExtKAS()
+    })
+
+    afterEach(() => {
+        sandbox.restore()
+    })
+
+    context('caver.initMetaDataAPI', () => {
+        it('CAVERJS-EXT-KAS-MATADATA-001: should return error if metadataAPI is not initialized', async () => {
```

Comment:
@richetoh Could you review if these tests can cover full functionalities of the metadata services?
@hyeKwon If you have some comments or feedback, please leave them.